pkg/operator/ceph/controller/controller_utils.go

```diff
@@ -133,6 +137,27 @@ func EnforceHostNetwork() bool {
 	return cephv1.EnforceHostNetwork()
 }
 
+func SetRevisionHistoryLimit(data map[string]string) {
+	strval := k8sutil.GetValue(data, revisionHistoryLimitSettingName, revisionHistoryLimitDefaultValue)
+	var numval int64 = 0
+	if strval != "" {
+		var err error
+		numval, err = strconv.ParseInt(strval, 10, 32)
+		if err != nil {
+			logger.Warningf("failed to parse value %q for %q. assuming default value 3.", strval, revisionHistoryLimitSettingName)
```

Comment:
we are not going to have default `3` in the code right? we need to update the warning message

Reply:
default reset to empty string and warning message adjusted. But now my unit test has a problem.